Clustering/src/clusterer.cxx

```diff
@@ -45,25 +80,26 @@ void Clusterer::Clustering(const char *inputFile) {
  * cluster is initialised with the hits found. After a cluster is found, it
  * moves on the following hit (not in the cluster) and repeats.
  */
-void Clusterer::EventClustering(FileManager &inputData, const int ievent) {
-
+std::vector<Cluster> Clusterer::EventClustering(FileManager &inputData,
```

Comment:
Is this really needed? You never call the Event Clustering to get the vector of clusters right?

Reply:
you're right I forgot  to change it